static/js/index.js

```diff
@@ -5,7 +5,10 @@ function renderGitHubCommits() {
     const date = new Date(commit["date"]+" UTC");
     var options = { year: 'numeric', 
                       month: 'long', 
-                      day: 'numeric'
+                      day: 'numeric',
+                      hour: 'numeric',
+                      minute: '2-digit',
+                      second: '2-digit',
                     };
     const strDate = date.toLocaleDateString("en-US", options)
```

Comment:
consider also using `toLocaleString("en-US")`, which gives reasonable output even without an options argument.

Reply:
actually we should probably use `toLocaleString` regardless, since it's not obvious that `toLocaleDateString` includes the time.

Reply:
just fixed